test/pylib/cpp/item.py

```diff
@@ -122,15 +123,15 @@ class CppFile(pytest.File):
     """
     def __init__(self, *, no_parallel_run: bool = False, modes: list[str], disabled_tests: dict[str, set[str]],
                  run_id=None, facade: CppTestFacade, arguments: Sequence[str], parameters: list[str] = None, project_root: Path,
-                 **kwargs: Any) -> None:
+                 env: dict = None, **kwargs: Any) -> None:
         super().__init__(**kwargs)
         self.facade = facade
         self.modes = modes
         self.run_id = run_id
         self.disabled_tests = disabled_tests
         self.no_parallel_run = no_parallel_run
         self.parameters = parameters
-        self.project_root = project_root
+        self.env = env
```

Comment:
I don't get this change. The CppFile class still as the `project_root` constructor argument, but it's now dropped on the floor and `self.project_root` no longer exists. At the same time the `CppFil::collect()` method still uses it after this change. What's going on here?